lib/tasks_private/terraform/provision.rake

```diff
@@ -0,0 +1,91 @@
+require "json"
+require "byebug"
```

Comment:
I realize this is WIP but ultimately, this bye bug and the one on line 5 will have to go away